index.js

```diff
@@ -0,0 +1,34 @@
+const API_KEY = `3265874a2c77ae4a04bb96236a642d2f`;
```

Comment:
A good practice is to move sensitive information to separate file, such as `config.js` and import them here to use the API_KEY and API_URL.